_unittest/test_03_Materials.py

```diff
@@ -363,6 +363,13 @@ def test_14_set_core_loss(self):
                 150: [[0, 0], [1, 10], [2, 19]],
             }
         )
+        try:
+            self.aedtapp.materials["mat_test"].set_coreloss_at_frequency(
+                points_list_at_freq={80: [[0, 0], [1, 3.5], [2, 7.4]]}, core_loss_model_type="Power Ferrite"
+            )
+            assert False
+        except ValueError:
+            assert True
```

Comment:
```suggestion
        with pytest.raises(ValueError):
            self.aedtapp.materials["mat_test"].set_coreloss_at_frequency(
                points_list_at_freq={80: [[0, 0], [1, 3.5], [2, 7.4]]}, core_loss_model_type="Power Ferrite"
            )
```

Reply:
I would recommend using pytest `raises` to ensure that an exception of a specific type is raised.
Btw, I can see a few lines before the following code :
```python
            assert False
        except TypeError:
            assert True
```
I can't understand why such lines are here. If you agree with the proposed modification, it would be great to also use `raises` in that case and to remove the assert `False` and `True`.